lib/matplotlib/quiver.py

```diff
@@ -585,7 +585,7 @@ def set_UVC(self, U, V, C=None):
         if C is not None:
             C = ma.masked_invalid(C, copy=True).ravel()
         for name, var in zip(('U', 'V', 'C'), (U, V, C)):
-            if var is not None and var.size != self.N:
+            if var is not None and var.size != self.N and var.size != 1:
```

Comment:
Maybe can this be rewritten as 
`if not (var is None or var.size == self.N or var.size==1):`  ?  I find three "nots" in a row starts to hurt my little head.

Reply:
done